doc/reference/tt_cli/start.rst

```diff
@@ -0,0 +1,53 @@
+.. _tt-start:
+
+Starting a Tarantool instance
+=============================
+
+..  code-block:: bash
+
+    tt start INSTANCE
+
+``tt start`` starts the specified Tarantool :ref:`instance <admin-instance_file>`.
+The specified ``INSTANCE`` value is used as this instance's identifier.
+
+Details
+-------
+
+The ``INSTANCE`` argument must specify a path to the instance file:
+
+*   from the ``instances_available`` path specified in the
+    :ref:`tt configuration file <tt-config_file_app>`.
+*   from the ``tt`` :ref:`working directory <tt-config_modes>` if ``instances_available``
+    is not specified.
+
+..  important::
+
+  The ``INSTANCE`` value must not include the ``.lua`` file extension.
+
+The ``INSTANCE`` value can be a directory name. In this case, ``tt`` starts
+the ``init.lua`` instance from this directory.
+
+Examples
+--------
+
+All paths in the examples below are relative to the ``tt`` working directory or
+``instances_available`` directory if it is specified.
+
+*   Start the ``app.lua`` instance:
+
+    ..  code-block:: bash
+
+        tt start app
+
+*   Start the ``app.lua`` instance from the ``instances`` directory:
+
+    ..  code-block:: bash
+
+        tt start instances/app
+
+
+*   Start the ``init.lua`` instance from the ``app/instance1/`` directory:
+
+    ..  code-block:: bash
+
+        tt start app/instance1
```

Comment:
This is not how it should work (only the name should be specified). If it works that way, it's a bug.

Reply:
Same here, it works:
```
p.semyonov@p-semyonov tt % ./tt check instances/instance1 
   • Result of check: syntax of file '/Users/p.semyonov/tt/instances/instance1/init.lua' is OK
p.semyonov@p-semyonov tt % ./tt start instances/instance1
p.semyonov@p-semyonov tt %    • Starting an instance...  

p.semyonov@p-semyonov tt % ./tt status instances/instance1
   • RUNNING. PID: 322.       
p.semyonov@p-semyonov tt % ./tt stop instances/instance1
2022/07/14 17:51:47 The Instance (PID = 322) has been terminated.
```